src/views/ManageList.jsx

```diff
@@ -24,21 +29,44 @@ export function ManageList({ listPath }) {
 		}
 	};
 
+	const handleShareSubmit = async (event) => {
+		event.preventDefault();
+		// shareList(listPath, currentUserId, recipientEmail)
+		try {
+			//if the user hasn't clicked on a list (state == null)
+			if (!listPath) {
+				alert('Please select a list to share.');
+				return;
+			}
+
+			//Future validation: Check for existence of the list already in recipient's collection.
+
+			const response = await shareList(listPath, userId, shareEmail);
```

Comment:
issue: When I tried a test that was supposed to work (i.e., inviting an existing user to a list that I own) I got the "List could not be created" alert instead of the success alert even though when I checked Firebase the list was successfully shared. I think it's because the ```shareList``` function isn't currently returning anything, so ```response``` is never truthy.

Reply:
Thanks for the catch! I've added a return true statement to the shareList function and it seems to be working on my end now.

Reply:
We did some more refactoring during office hours! 😄